internal/archiver/archiver.go

```diff
@@ -791,6 +795,15 @@ func (arch *Archiver) Snapshot(ctx context.Context, targets []string, opts Snaps
 		return nil, restic.ID{}, err
 	}
 
+	if opts.Command != nil {
+		errBytes, _ := io.ReadAll(opts.CommandStderr)
```

Comment:
This will deadlock if a process writes too much data to stderr. Please handle stderr similar as in the rclone/sftp backend.